### PR TITLE
fix(core/inlines): expand markdown headings

### PR DESCRIPTION
**Pull request by sidvishnoi**
`core/inlines` was ignoring the fact the `core/id-headers` adds IDs after it.
See https://github.com/w3c/spec-prod/pull/69#discussion_r641444950

### CODE REVIEW

File: src/core/inlines.js [hunk 1]
```diff
@@ -119,14 +118,8 @@ function inlineRefMatches(matched) {
...
   }
-  if (document.querySelector(ref)) {
-    return html`<a href="${ref}"></a>`;
-  }
-  const badReference = html`<span>${matched}</span>`;
-  const msg = `Wasn't able to expand ${matched} as it didn't match any id in the document.`;
-  const hint = `Please make sure there is element with id ${ref} in the document.`;
-  showError(msg, name, { hint, elements: [badReference] });
-  return badReference;
+  console.log(matched, ref);
```
Comment (marcoscaceres) [inline src/core/inlines.js:121]: Stray console.log? 
```suggestion
```

File: tests/spec/core/inlines-spec.js [hunk 2]
```diff
@@ -245,18 +249,22 @@ describe("Core - Inlines", () => {
         <aside class="example" id="example-aside_thing" title="aside"></aside>
         <pre class="example" id="example-pre" title="pre">
         </pre>
+        <section data-include="${generateDataIncludeUrl(
```
Comment (marcoscaceres) [inline tests/spec/core/inlines-spec.js:252]: Cute solution!

**Review by marcoscaceres — APPROVED** [review]
Looks great! And it gets rid of duplicate error message! Woot Woot!

File: tests/spec/core/inlines-spec.js [hunk 3]
```diff
@@ -245,18 +249,22 @@ describe("Core - Inlines", () => {
...
         [[[#example-aside_thing]]]
         [[[#example-pre]]]
+        [[[#example-dynamic]]]
         [[[#does-not-exist]]]
       </p>`;
     const doc = await makeRSDoc(makeStandardOps(null, body));
     const anchors = doc.querySelectorAll("#output a");
-    expect(anchors).toHaveSize(4);
-    const [section, figure, exampleAside, examplePre] = anchors;
+    expect(anchors).toHaveSize(6);
```
Comment (marcoscaceres) [inline tests/spec/core/inlines-spec.js:266]: What's the 6th one?

Comment (sidvishnoi) [reply tests/spec/core/inlines-spec.js:266]: `#does-not-exist`. Earlier it was converted into a span (in core/inlines).

File: tests/spec/core/inlines-spec.js (hunk 2 above)
Comment (sidvishnoi) [reply tests/spec/core/inlines-spec.js:252]: Copied in from data-include tests 😁

**Pull request merged**